csm_web/scheduler/models.py

```diff
@@ -66,6 +72,22 @@ class Meta:
         abstract = True
 
 
+class Semester(ValidatingModel):
+    sid = models.PositiveSmallIntegerField(primary_key=True)
+    objects = models.Manager()
+
+    def __str__(self):
+        return f"{self.sid}"
```

Comment:
If you do end up adding the extra description field, that should be added here to describe the semester model a little bit more.